lib/index.js

```diff
@@ -393,7 +451,7 @@ function morphdom(fromNode, toNode, options) {
             if (toNodeType === 1) {
                 if (fromNode.tagName !== toNode.tagName) {
                     onNodeDiscarded(fromNode);
-                    morphedNode = moveChildren(fromNode, document.createElement(toNode.tagName));
+                    morphedNode = moveChildren(fromNode, createElement(toNode.tagName, toNode.namespaceURI));
```

Comment:
For better browser support it's adviced to use nodeName instead of tagName
tagName is DOM Level 3 Element. NodeName is part of Node of Dom Level 2 which has better support among older browsers